Assets/Scripts/Towers/Specific towers/Axe turret/AxeTowerAnimationController.cs

```diff
@@ -21,7 +21,7 @@ public void Grab()
     public void Shoot()
     {
         handAxe.enabled = false;
-        Rigidbody projectileBody = Instantiate(axeProjectilePrefab, spawnPoint.position, spawnPoint.rotation)
+        Rigidbody projectileBody = Instantiate(axeProjectilePrefab, spawnPoint.position, spawnPoint.rotation, gameObject.transform)
```

Comment:
You don't have to get `transform` through the `gameObject`, as it's an available property through `MonoBehaviour` 🤠
```suggestion
        Rigidbody projectileBody = Instantiate(axeProjectilePrefab, spawnPoint.position, spawnPoint.rotation, transform)
```